pkg/controller/plan/adapter/ova/builder.go

```diff
@@ -517,51 +518,36 @@ func getDiskSourcePath(filePath string) string {
 }
 
 func getResourceCapacity(capacity int64, units string) (int64, error) {
-	if units == "" || units == "byte" {
-		return capacity, nil
-	}
-
-	numbers, err := extractNumbers(units)
-	if err != nil || len(numbers) < 2 {
-		return 0, err
-	}
-
-	pow, err := strconv.Atoi(numbers[len(numbers)-1])
-	if err != nil {
-		return 0, err
-	}
-
-	base, err := strconv.Atoi(numbers[len(numbers)-2])
-	if err != nil {
-		return 0, err
-	}
-
-	result := capacity
-	for _, num := range numbers[:len(numbers)-2] {
-		val, err := strconv.Atoi(num)
-		if err != nil {
-			return 0, err
+	items := strings.Split(units, "*")
```

Comment:
looking good, the only thing I"ll add is a check if its not an empty string in the beginning of the func and if so return the capacity.
+ if the tests passed it should be verified.

Reply:
> looking good, the only thing I"ll add is a check if its not an empty string in the beginning of the func and if so return the capacity.

that's what the first test checks, right? :)
it is supposed to work even without special handling to this case